site.yml

```diff
@@ -28,8 +28,8 @@
     - docker
     - weave
     - dnsmasq
-    - registrator
     - cadvisor
+    - registrator
```

Comment:
Should be irrelevant as we are getting the ip in your PR via docker inspect, so all we need to do when merging both is making sure your "docker inspect"  goes through the weave proxy.